assignment4.Rmd

```diff
@@ -105,12 +111,41 @@ sweep(loadings, 2, colSums(loadings), "/") #sweep() computes each row as a propo
 
 #Now examine your components and try to come up with substantive descriptions of what some might represent?
 
+#PC1 explains the largest proportion of variance on two variables: mean_hint(30%) and mean_attempt(25%); For PC2, it accounts for the largest proportion of variance on prior_percent_correct(45%); PC3 contributes its higher share of var into mean_confidence(46%); For PC4, prior_prob_count has the highest proportion of variance (32%); For PC5, mean_attempt(36%) and problems_attempted(30%) take up bigger proportions of var; and PC6 contributes the most variance into mean_hint(36%).
```

Comment:
Good work interpreting what each component means. If we take it a little further, we can assign ideas to what each component represents. An example of this would be mean_hint and mean_attempt are well represented by PC1, so maybe it represents their 'aggressiveness' at clicking.